### ISSUE TITLE
Bug - Weight and slots does not change even if you configure it

### ISSUE THREAD

**Issue by LorKami** [Ranjit-Develops/Ranjit-EMS-bag#8]
https://i.imgur.com/p5kMygC.png
https://i.imgur.com/YbrAdnc.png

Even if you change it in the configuration, it has no effect.

**Comment by nzkfc** [comment]
Yeah even setting it manually doesn't work, I think it's the way it has been coded is diff to how it should be


RegisterNetEvent('qb-traphouse:client:target:ViewInventory', function (data)
    local TraphouseInventory = {}
    TraphouseInventory.label = "traphouse_"..CurrentTraphouse
    TraphouseInventory.items = data.traphouseData.inventory
    TraphouseInventory.slots = 2
    TriggerServerEvent("inventory:server:OpenInventory", "traphouse", TraphouseInventory.label, TraphouseInventory)
end)
